X10D/src/Collections/DictionaryExtensions.cs

```diff
@@ -51,9 +119,9 @@ public static class DictionaryExtensions
         }
 #endif
 
-        if (dictionary.ContainsKey(key))
+        if (dictionary.TryGetValue(key, out var old))
```

Comment:
Since the assigned type here is not inherently apparent, explicitly declaring `old` as `TValue` is preferred.